docker-compose-dev-linux.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
```

Comment:
This probably should be in a separate PR.

Reply:
https://github.com/ONSdigital/eq-survey-runner/pull/2099